Common/yaml/Contacts.yaml

```diff
@@ -0,0 +1,374 @@
+  version: 1.0.11
+externalDocs:
+  description: Find out more about Smart Contact Manager
+  url: http://swagger.io
+servers:
+  - url: http://localhost:8082
+tags:
+  - name: contacts
+    description: Apis related to contacts
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+  - name: categories
+    description: Apis related to categories
+    externalDocs:
+      description: Find out more about our store
+      url: http://swagger.io
+  - name: user
+    description: Operations about user
+paths:
+  /v1/contacts:
+    post:
+      tags:
+        - contacts
+      summary: Add a new contat
+      description: Add a new contact
+      operationId: addContact
+      requestBody:
+        description: Create a new contact
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Contact'
+      responses:
+        '201':
+          description: Successful
+          content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: "Contact Created Successfully"
+        '400':
+          description: BAD REQUEST
+          content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: "Invalid Contact Details"
+        '500':
+            description: Internal Server Error
+            content:
+              application/json:
+                schema:
+                  type: string
+                  example:
+                    message: "Something went Wrong"
+        
+    delete:
+      tags:
+        - contacts
+      summary: Delete all contacts
+      description: Delete all contacts
+      operationId: deleteContacts
+      responses:
+        '200':
+          description: Successful
+          content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: "All Contacts Deleted Successfully"
+        '500':
+            description: Internal Server Error
+            content:
+              application/json:
+                schema:
+                  type: string
+                  example:
+                    message: "Something went Wrong"
+    get:
+     
+      tags:
+        - contacts
+      summary: Get all contacts
+      description: Get all contacts
+      operationId: Get contacts
+      parameters:
+        - name: page
+          in: query
+          description: page value for pagination
+          required: false
+          schema:
+            type: integer
+            example: 0
+        - name: limit
+          in: query
+          description: limit of records in one page for pagination
+          required: false
+          schema:
+            type: integer
+            example: 3 
+        - name: name
+          in: query
+          description: Enter the Name
```

Comment:
Correct the description here and below params

Reply:
fixed please review